src/main/java/com/jalasoft/webservice/model/OrcConvert.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+
+package com.jalasoft.webservice.model;
+
+import com.jalasoft.webservice.entitities.ModelFile;
+import com.jalasoft.webservice.entitities.OrcFile;
+import com.jalasoft.webservice.entitities.TextFile;
+import com.jalasoft.webservice.utils.ConfigurationVariable;
+import com.jalasoft.webservice.utils.Constants;
+import com.jalasoft.webservice.utils.PropertiesReader;
+import net.sourceforge.tess4j.Tesseract;
+import net.sourceforge.tess4j.TesseractException;
+
+import java.io.File;
+
+import static com.jalasoft.webservice.utils.Constants.APPLICATION_PROPERTIES;
+
+public class OrcConvert implements IConvert{
+    private PropertiesReader propertiesFile = new PropertiesReader("src/main/resources/", APPLICATION_PROPERTIES);
+    @Override
+    public ModelFile Convert(ModelFile model) {
+        String tesseractPath = "file.tesseract-path";
```

Comment:
move this variables outside of the method as private class variable:

```java
        String tesseractPath = "file.tesseract-path";
        String defaultLanguageCode = "file.default-languageCode";
        String defaultLanguage = "file.default-language";
```

Reply:
done